src/Rubric/RubCharacterScanner.class.st

```diff
@@ -85,7 +85,6 @@ RubCharacterScanner >> basicScanCharactersFrom: startIndex to: stopIndex in: sou
 	reached, then return stops at: 257. Optional.
 	See Object documentation whatIsAPrimitive."
 	| ascii nextDestX char floatDestX widthAndKernedWidth nextChar atEndOfRun |
-	<primitive: 103>
```

Comment:
Is it normal to remove the call to a primitive and only keep the fallback code?

Reply:
See the ‘pharo-vm’ commit referred to in commit bb13a55ff2ba9c52: https://github.com/pharo-project/pharo-vm/commit/b80f660c5c9b1bbba5bc. In the primitive table, 103 was changed to `#primitiveFail`.

Reply:
Hi kris all your work is super nice :)
I hope that you will come to the sprint at the end of the month. 
Do you have a code snippet to enable this nice looking changes on our poor Retina screens :)
I know it should be somewhere but I do not remember.

Reply:
Thanks! I will put the snippet in a separate comment below.